node/db/src/parity_db.rs

```diff
@@ -154,3 +169,27 @@ impl BitswapStore for ParityDb {
 }
 
 impl DBStatistics for ParityDb {}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use parity_db::CompressionType;
+
+    #[test]
+    fn compression_type_from_str_test() {
+        let test_cases = vec![
```

Comment:
static array should do fine
```suggestion
        let test_cases = [
```

Reply:
alright